includes/payment-methods/class-upe-payment-gateway.php

```diff
@@ -875,6 +879,30 @@ public function clear_upe_appearance_transient() {
 		delete_transient( self::UPE_APPEARANCE_TRANSIENT );
 	}
 
+	/**
+	 * Sets the title on checkout correctly before the title is displayed.
+	 *
+	 * @param {string} $title The title of the gateway being filtered.
```

Comment:
Nitpick: The format for docblocks is slightly different from what I've seen elsewhere in the codebase, perhaps we should update it here to ensure it is standardised? This is what I normally see (the only difference being the lack of `{ }` surrounding the type. I think that may be a JS convention!

```
* @param string $title The title of the gateway being filtered
```

Reply:
I noticed the different format and copied it rather than asking about if they should be changed. Thanks for the nitpick, I just updated them. https://github.com/Automattic/woocommerce-payments/pull/3335/commits/fa7b2f00c0132107d75941442de0565889424af7